### PR TITLE
sort terraform dependencies

### PR DESCRIPTION
**Pull request by hugoesthere**
Resolves https://github.com/Originate/exosphere/issues/901

### CODE REVIEW

File: src/config/config_helpers.go [hunk 1]
```diff
@@ -55,3 +56,18 @@ func GetAllRemoteDependencies(appContext *context.AppContext) map[string]types.R
...
+		result = append(result, k)
+	}
+	for _, serviceContext := range appContext.ServiceContexts {
+		for k := range serviceContext.Config.Remote.Dependencies {
+			result = append(result, k)
+		}
+	}
+	sort.Strings(result)
+	return result
+}
```
Comment (charlierudolph) [inline src/config/config_helpers.go]: I think this should be a method on `appContext` since that is the only argument.

**Review by charlierudolph — APPROVED** [review]
Good to merge after moving the method

File: src/config/config_helpers.go (hunk 1 above)
Comment (hugoesthere) [reply src/config/config_helpers.go]: There are also a number of other methods that only rely on `appContext`. Should we move those as well? I think we need to also redefine what this `config` package is used for/what we should name it. I don't think it serves the purpose of config helpers anymore and `config` always felt like a vague package name to me anyways

Comment (charlierudolph) [reply src/config/config_helpers.go]: For now, can we move just this method and then refactor the config package in another PR. I'll create an issue where we can discuss where to move the methods

**Pull request merged**